assets/js/vue/components/search/ResultRow.vue

```diff
@@ -1,25 +1,38 @@
 <template>
-    <b-card class="card-product" no-body>
+    <b-card class="card-product">
         <b-link class="search-result"
                 :href="url" target="_blank">
-            <b-card-body>
-                <div class="row">
-                    <article class="col-lg-12">
-                        <span class="badge badge-secondary" v-if="resultRowData.availabilityStatus === 0">Identified</span>
-                        <span class="badge badge-primary" v-else-if="resultRowData.availabilityStatus === 2 || resultRowData.availabilityStatus === 4">Submitted</span>
-                        <span class="badge badge-danger" v-else-if="resultRowData.availabilityStatus === 5 || resultRowData.availabilityStatus === 8">Restricted</span>
-                        <span class="badge badge-success" v-else-if="resultRowData.availabilityStatus === 7 || resultRowData.availabilityStatus === 10">Available</span>
-                        <b-card-text>
-                            {{ resultRowData.datasetSubmission.authors }}.
-                            {{ resultRowData.year }}.
-                            {{ resultRowData.title }}.
-                            {{ resultRowData.researchGroup.name }}.
-                            DOI: {{ resultRowData.doi.doi }}.
-                            UDI: {{ resultRowData.udi }}
-                        </b-card-text>
-                    </article>
+            <div>
+                <span class="badge badge-secondary" v-if="resultRowData.availabilityStatus === 0">Identified</span>
+                <span class="badge badge-primary" v-else-if="resultRowData.availabilityStatus === 2 || resultRowData.availabilityStatus === 4">Submitted</span>
+                <span class="badge badge-danger" v-else-if="resultRowData.availabilityStatus === 5 || resultRowData.availabilityStatus === 8">Restricted</span>
+                <span class="badge badge-success" v-else-if="resultRowData.availabilityStatus === 7 || resultRowData.availabilityStatus === 10">Available</span>
+            </div>
+            <b-card-title>{{ resultRowData.title }}</b-card-title>
+            <b-card-text class="d-flex justify-content-between" >
+                <div v-if="Object.keys(resultRowData.datasetSubmission).length > 0">
+                    <div v-if="resultRowData.datasetSubmission.authors">
```

Comment:
I would wrap these in a div maybe, with a style. Font could be a bit smaller for more real estate, but that's just my opinion. No change needed.